contract_sale/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Contract from Sale",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
```

Comment:
This change shouldn't go here, but in the `[MIG]` commit.

Reply:
In the pre-commit it shows error if we change it in migration commit.

Reply:
Yes, that is normal, that is why we do the first `[IMP]` with `--no-verify`, to avoid those. When you change it latter in the `[MIG]` commit it will not show the error anymore.